thirdparty/cmdconfig/commands/fnevalcmd.go

```diff
@@ -101,16 +75,16 @@ type RunFnRunner struct {
 	AsCurrentUser      bool
 }
 
-func (r *RunFnRunner) runE(c *cobra.Command, args []string) error {
+func (r *EvalFnRunner) runE(c *cobra.Command, args []string) error {
 	return runner.HandleError(c, r.RunFns.Execute())
 }
 
 // getContainerFunctions parses the commandline flags and arguments into explicit
 // Functions to run.
-func (r *RunFnRunner) getContainerFunctions(c *cobra.Command, dataItems []string) (
+func (r *EvalFnRunner) getContainerFunctions(dataItems []string) (
```

Comment:
One observation:
The way this code is written is to have a common code for function invocation with declarative/imperative runs. For example, it creates a functionConfig for `image` or `execPath`. Given that we are dropping declarative runs, we can probably simplify this a lot more.

Reply:
Yes I agree

Reply:
We don't need to address this immediately, but something to keep in mind. if we end up in situation where lot of changes need to be made to this code, we can then simplify it.